forge-gui/res/cardsfolder/a/adaptive_sporesinger.txt

```diff
@@ -7,5 +7,6 @@ T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.S
 SVar:TrigCharm:DB$ Charm | Choices$ DBPump,DBProliferate
 SVar:DBPump:DB$ Pump | NumAtt$ 2 | NumDef$ 2 | ValidTgts$ Creature | KW$ Vigilance | SpellDescription$ Target creature gets +2/+2 and gains vigilance until end of turn.
 SVar:DBProliferate:DB$ Proliferate | SpellDescription$ Proliferate. (Choose any number of permanents and/or players, then give each another counter of each kind already there.)
-DeckHas:Ability$Proliferate
+DeckHas:Ability$Proliferate|Counters
```

Comment:
this card doesn't have a "Counters" ability by itself

Reply:
imo yes, I can see why it could be interpreted that way but remember it's mostly for deckbuilding

e.g. _Bloom Hulk_ gains nothing if you'd encourage a combination with this one